src/meson.build

```diff
@@ -15,6 +14,7 @@ libekb = meson.get_compiler('cpp').find_library('ekb')
 
 libedbg = library(
             'edbg',
+            'dll/edbgEcmdDllScom_p9.C',
```

Comment:
Would it make more sense to put these files in the p9 directory?

Reply:
file name already has _p9.C. isn't that good?

Reply:
Why have a p9, etc.. directory if the p9, etc.. specific code isn't in it?

Reply:
Done. move to p9 folder